docs/source/eventing/actions/label.rst

```diff
@@ -20,17 +20,25 @@ Changes the text and color of a generic label device.
 +----------+------------------+---------------------+---------------------------------------------------+
 | TO       | |yes|            | |no|                | New text                                          |
 +----------+------------------+---------------------+---------------------------------------------------+
-| COLOR    | |no|             | |no|                | New color. If omitted, color is set to black"     |
-+----------+------------------+---------------------+---------------------------------------------------+
 | **Name** | **Required**     | **Multiple Values** | **Description**                                   |
 +----------+------------------+---------------------+---------------------------------------------------+
+| COLOR    | |no|             | |no|                | New color                                         |
++----------+------------------+---------------------+---------------------------------------------------+
+| BLINK    | |no|             | |no|                | Blinking on or off                                |
```

Comment:
Can you change this in a `.. versionadded:: nightly` approach?

Reply:
Done